### ISSUE TITLE
Port Qt backend to Qt 6

### ISSUE THREAD

**Issue by jgcodes2020** [slint-ui/slint#6586]
The Qt backend should be ported to Qt 6 ASAP, as the current backend runs on Qt 5, set to be [discontinued May 2025](https://www.qt.io/blog/qt-5.15-extended-support-for-subscription-license-holders).

**Comment by ogoffart** [comment]
The Qt backend is already ported to Qt6.
It even defaults to Qt6 if both Qt6 and Qt5 are installed.

Although I realized that the linux package are still built with Qt5.
That was because on Linux, most distribution still used KDE that was using Qt5. But I guess now, we should change the package to be built with Qt6.

**Comment by jgcodes2020** [comment]
I did a bit more digging in the code; seems to be related to https://github.com/woboq/qmetaobject-rs/issues/314. I've submitted a PR that should ensure `qttypes` defaults to Qt 6 where possible.

**Comment by ogoffart** [comment]
Indeed, i was thinking we used Qt6 by default but that's only in CMake with C++. 
With Rust, qttypes indeed defaulted to whatever 'qmake' is which is most likely Qt5.  New release of qttypes was released

**Comment by JoeySoprano420** [comment]
Unsubscribr
